controllers/__init__.py

```diff
@@ -0,0 +1,126 @@
+"""MaintenanceController."""
+
+# pylint: disable=invalid-name
+from datetime import datetime
+import os
+import pytz
+from typing import Optional
+
+from bson.codec_options import CodecOptions
+import pymongo
+from pymongo.errors import AutoReconnect
+from tenacity import retry_if_exception_type, stop_after_attempt, wait_random
+from werkzeug.exceptions import BadRequest
+
+from kytos.core import log
+from kytos.core.db import Mongo
+from kytos.core.retry import before_sleep, for_all_methods, retries
+from napps.kytos.maintenance.models import (
+    MaintenanceWindow,
+    MaintenanceWindows,
+    MaintenanceID,
+    Status,
+)
+
+
+@for_all_methods(
+    retries,
+    stop=stop_after_attempt(
+        int(os.environ.get("MONGO_AUTO_RETRY_STOP_AFTER_ATTEMPT", 3))
+    ),
+    wait=wait_random(
+        min=int(os.environ.get("MONGO_AUTO_RETRY_WAIT_RANDOM_MIN", 0.1)),
+        max=int(os.environ.get("MONGO_AUTO_RETRY_WAIT_RANDOM_MAX", 1)),
+    ),
+    before_sleep=before_sleep,
+    retry=retry_if_exception_type((AutoReconnect,)),
+)
+class MaintenanceController:
```

Comment:
@Ktmi this module and class hasn't been covered in unit tests, the total coverage fell to 62%, we're trying to maintain at least 85%. 

Currently, we've been unit testing DB controllers this way, examples below:

https://github.com/kytos-ng/flow_manager/blob/master/tests/unit/test_flow_controller.py
https://github.com/kytos-ng/topology/blob/master/tests/unit/test_topo_controller.py

Reply:
With the latest commits, this pendency was fixed. According to [Scrutinizer's last execution](https://scrutinizer-ci.com/g/kytos-ng/maintenance/inspections/6c472997-3965-4cf2-81e0-a1e65329d306), code coverage increased to 87%! Very nice job, @Ktmi!